x/txfees/keeper/txfee_filters/arb_tx_test.go

```diff
@@ -65,6 +65,36 @@ func (suite *KeeperTestSuite) TestIsArbTxLooseAuthz_AffiliateSwapMsg() {
 	suite.Require().True(isArb)
 }
 
+// Tests that the arb filter is enabled on swap msg.
+func (suite *KeeperTestSuite) TestIsArbTxLooseAuthz_SwapMsg() {
+	contractSwapMsg := &txfee_filters.ContractSwapMsg{
+		ContractSwap: txfee_filters.ContractSwap{
+			InputCoin: txfee_filters.InputCoin{
+				Amount: "2775854",
+				Denom:  "ibc/D1542AA8762DB13087D8364F3EA6509FD6F009A34F00426AF9E4F9FA85CBBF1F",
+			},
+			OutputDenom: "ibc/D1542AA8762DB13087D8364F3EA6509FD6F009A34F00426AF9E4F9FA85CBBF1F",
+			Slippage: txfee_filters.Slippage{
```

Comment:
Some of the contracts allow to set slippage as a percentage instead of MinOutputAmount. Maybe we should consider that as an option too?